### PR TITLE
Replace datetime calls with time.time equivalents.

### PR DESCRIPTION
**Pull request by arjclark**
Update the time calculations used for periodic checking of file system to use time.time rather than datetime as recommended in #1518 

@matthewrmshin  - please review this minor change as a speedup of the existing code.

### CODE REVIEW

File: lib/cylc/scheduler.py [hunk 1]
```diff
@@ -906,8 +906,8 @@ def run(self):
 
         proc_pool = SuiteProcPool.get_inst()
 
-        fs_check_period = datetime.timedelta(minutes=10)
-        next_fs_check = datetime.datetime.utcnow() + fs_check_period
+        fs_check_period = 600.0
```
Comment (matthewrmshin) [inline lib/cylc/scheduler.py]: Given the nature of this variable, you may want to put this at the top of the class/module, and mark it as a class/module level constant. It is also worth adding a trailing `# seconds` comment to indicate that we are talking about 600.0 seconds.

Comment (arjclark) [inline lib/cylc/scheduler.py]: Done.

**Comment by arjclark** [comment]
@matthewrmshin - changed as suggested.

**Comment by matthewrmshin** [comment]
@hjoliver please review 2.

**Comment by hjoliver** [comment]
Good.

**Pull request merged**